src/test/kotlin/no/nav/syfo/aktivitetskrav/database/AktivitetskravRepositorySpek.kt

```diff
@@ -74,6 +77,59 @@ class AktivitetskravRepositorySpek : Spek({
                     newVarselPdf?.pdf?.get(1) shouldBeEqualTo pdf[1]
                     newVarselPdf?.aktivitetskravVarselId shouldBeEqualTo newVarsel.id
                 }
+
+                it("Should retrieve expired varsler correctly") {
```

Comment:
Fin test :partying_face: Hadde vært bra om it-en har sa noe om hva som er correctly (de med frist oneWeekAgo eller eldre) så ser man det litt tydelig når testene kjøres.

Reply:
Ja, lurt🙌 Legger det til